examples/custom_snippet.py

```
@@ -0,0 +1,94 @@
+    name = Column(String)
+    email = Column(String, unique=True)
+    website = Column(String)
+
+    class Configuration:
+        http_method_names = ['GET', 'POST']
+        validator_class = CustomEndpointValidator
+        filter_class = ParameterFilter
+
+    def post(self, request):
+        return Response(
+            {'data': 'ok', 'data': getattr(request, 'data', {})},
```

Ruff (0.11.9)

Dictionary key literal `'data'` repeated

(F601)